ipaserver/install/certs.py

```diff
@@ -360,14 +360,14 @@ def create_server_cert(self, nickname, hostname, subject=None):
         if subject is None:
             subject=DN(('CN', hostname), self.subject_base)
         self.request_cert(subject, san_dnsnames=[hostname])
-        self.issue_server_cert(self.certreq_fname, self.certder_fname)
-        self.import_cert(self.certder_fname, nickname)
-        fd = open(self.certder_fname, "r")
-        dercert = fd.read()
-        fd.close()
-
-        os.unlink(self.certreq_fname)
-        os.unlink(self.certder_fname)
+        try:
+            cdb.issue_server_cert(self.certreq_fname, self.certder_fname)
```

Comment:
The behavior here changed a bit, there's no need to ask another database to issue server cert for us since the location of Dogtag agent certificates is know.
Do `s/cdb/self/`.